Dockerfile.assisted-service

```diff
@@ -25,7 +25,7 @@ RUN CGO_ENABLED=0 GOFLAGS="" GO111MODULE=on go build -o /build/assisted-service
 RUN cd build && python3 setup.py sdist --dist-dir /assisted-service-client
 
 # Create final image
-FROM centos:8
+FROM quay.io/generic/centos8
```

Comment:
I wonder if we can get a centos image hosted by some more official org.
I would say this is fine for now to get things green again, but it would be good to look into it.

Reply:
This image is also ~ 4 times as large as the one we were using before ... not sure what that's about :worried:

Reply:
@ronniel1 FYI

Reply:
@mhrivnak and I discussed this a bit in https://github.com/openshift/assisted-installer-agent/pull/48 but didn't come to any real conclusion.

Reply:
we have https://quay.io/repository/app-sre/centos that you can use for the interim until you switch to UBI